docs/concepts/Auditing-Packages.md

```diff
@@ -79,6 +79,18 @@ On NuGet.org, you can navigate to the package details page and click `Report pac
 If no security vulnerabilities are found, this means that packages with known vulnerabilities were not found in your package graph at the present moment of time you checked.
 Since the advisory database can be updated at any time, we recommend regularly checking your `dotnet restore` output and ensuring the same in your continuous integration process.
 
+## Configuring NuGet Audit
+
+### Source of vulnerability information
+
+NuGet will read [`nuget.config` files](../reference/nuget-config-file.md) from [multiple locations](../consume-packages/configuring-nuget-behavior.md#common-nuget-configurations).
+
+When at least one [audit source](../reference/nuget-config-file.md#auditsources) is provided, NuGet will use these `auditSources` to download a vulnerability database.
```

Comment:
We should probably add a note saying this is only available in VS17.11 and whatever version of the .NET SDK this corresponds to, since when the docs are published, most people won't be using an appropriate version yet.